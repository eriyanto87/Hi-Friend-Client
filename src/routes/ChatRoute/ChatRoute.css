.botchat {
  text-align: center;
  color: black;
  display: flex;
  justify-content: center;
  align-items: center;
}

.react-chatbot-kit-chat-header {
  background: #fc9f9f;
  border-bottom: 1px solid black;
  font-size: 20px;
}

.react-chatbot-kit-chat-bot-message {
  margin: 0px;
  border-radius: 18px;
  padding: 15px;
  background: lightgrey;
  border: 1px solid black;
  font-size: 18px;
}

.chatcomponent {
  text-align: center;
}

.react-chatbot-kit-chat-btn-send {
  margin-bottom: 0;
}

/* @media all and (min-width: 800px) {
  .react-chatbot-kit-chat-container {
    position: relative;
    width: 50%;
  }
} */
