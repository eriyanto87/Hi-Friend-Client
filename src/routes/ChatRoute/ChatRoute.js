import React from "react";
import Home from "../../components/Header/Header";
import Chat from "../../components/Chat/Chat";
import Footer from "../../components/Footer/Footer";
import "./ChatRoute.css";

export default function ChatRoute() {
  return (
    <>
      <Home />
      <Chat />
      <Footer />
    </>
  );
}
