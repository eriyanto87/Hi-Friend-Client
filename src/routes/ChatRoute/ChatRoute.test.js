import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import ChatRoute from "./ChatRoute";

describe("Chat Route", () => {
  it("ChatSnapshot", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <ChatRoute />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <ChatRoute />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });
});
