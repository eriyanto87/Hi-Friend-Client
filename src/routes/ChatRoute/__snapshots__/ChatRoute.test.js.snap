// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chat Route ChatSnapshot 1`] = `
Array [
  <div
    className="home"
  >
    <div>
      <a
        href="/"
      >
        Go To Home
      </a>
    </div>
  </div>,
  <h1
    className="chatcomponent"
  >
    Hi, 
  </h1>,
  <p
    className="chatcomponent"
  >
    <a
      href="/design"
      onClick={[Function]}
    >
      <button>
        Click here to name my chatbot
      </button>
    </a>
  </p>,
  <div
    className="botchat"
  >
    <div
      className="react-chatbot-kit-chat-container"
    >
      <div
        className="react-chatbot-kit-chat-inner-container"
      >
        <div
          className="react-chatbot-kit-chat-header"
        >
          Conversation with new best friend
        </div>
        <div
          className="react-chatbot-kit-chat-message-container"
        >
          <div
            style={
              Object {
                "paddingBottom": "15px",
              }
            }
          />
        </div>
        <div
          className="react-chatbot-kit-chat-input-container"
        >
          <form
            className="react-chatbot-kit-chat-input-form"
            onSubmit={[Function]}
          >
            <input
              className="react-chatbot-kit-chat-input"
              onChange={[Function]}
              placeholder="Write your message here"
              value=""
            />
            <button
              className="react-chatbot-kit-chat-btn-send"
              style={
                Object {
                  "backgroundColor": "#669db3ff",
                }
              }
            >
              <svg
                className="react-chatbot-kit-chat-btn-send-icon"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M476 3.2L12.5 270.6c-18.1 10.4-15.8 35.6 2.2 43.2L121 358.4l287.3-253.2c5.5-4.9 13.3 2.6 8.6 8.3L176 407v80.5c0 23.6 28.5 32.9 42.5 15.8L282 426l124.6 52.2c14.2 6 30.4-2.9 33-18.2l72-432C515 7.8 493.3-6.8 476 3.2z"
                />
              </svg>
            </button>
          </form>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="footer"
  >
    Â© 2020
  </div>,
]
`;
